docs/source/contributor-guide/tracing.md

```diff
@@ -43,10 +45,20 @@ Example output:
 { "name": "CometExecIterator_getNextBatch", "cat": "PERF", "ph": "B", "pid": 1, "tid": 5, "ts": 10109479156 },
 ```
 
-Traces can be viewed with [Trace Viewer]. 
+Traces can be viewed with [Trace Viewer].
 
 [Trace Viewer]: https://github.com/catapult-project/catapult/blob/main/tracing/README.md
 
 Example trace visualization:
 
 ![tracing](../_static/images/tracing.png)
+
+## Definition of Labels
+
+| Label                 | Meaning                                                        |
+| --------------------- | -------------------------------------------------------------- |
+| jvm_heapUsed          | JVM heap memory usage of live objects for the executor process |
+| jemalloc_allocated    | Native memory usage for the executor process                   |
+| task_memory_comet_NNN | Off-heap memory allocated by Comet for query execution         |
```

Comment:
what is NNN is it a task number or thread number or query number?

Reply:
Thread number

Reply:
> Thread number

it might be beneficial for the user to mention it